iceoryx_hoofs/test/moduletests/test_vocabulary_expected.cpp

```diff
@@ -477,6 +477,22 @@ TEST_F(expected_test, ErrorTypeOnlyMoveAssignmentLeadsToMovedSource)
     EXPECT_EQ(sutDestination.get_error().m_b, B);
 }
 
+TEST_F(expected_test, CreateFromInPlaceTypeLeadsToValidErrorOnlySut)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "91a8ad7f-4843-4bd9-a56b-0561ae6b56cb");
+    expected<TestError> sut{in_place};
+    ASSERT_THAT(sut.has_error(), Eq(false));
+}
+
+TEST_F(expected_test, CreateFromInPlaceTypeLeadsToValidSut)
```

Comment:
What about `CreateFromUnexpectTypeLeadsToInvalidSut`?

Reply:
There is not yet an `unexpect_t` available.